engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/immutable/UnsafeAtVectorNode.java

```diff
@@ -0,0 +1,41 @@
+package org.enso.interpreter.node.expression.builtin.immutable;
+
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.interop.InvalidArrayIndexException;
+import com.oracle.truffle.api.interop.UnsupportedMessageException;
+import com.oracle.truffle.api.library.CachedLibrary;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.epb.node.CoercePrimitiveNode;
+import org.enso.interpreter.node.expression.foreign.CoerceNothing;
+import org.enso.interpreter.runtime.Context;
+import org.enso.interpreter.runtime.data.Vector;
+import org.enso.interpreter.runtime.error.PanicException;
+
+@BuiltinMethod(
+    type = "Vector",
+    name = "unsafe_at",
+    description = "Returns an element of Vector at the specified index.")
+public class UnsafeAtVectorNode extends Node {
+
+  private @Child CoercePrimitiveNode coercePrimitiveNode = CoercePrimitiveNode.build();
+  private @Child CoerceNothing coerceNothingNode = CoerceNothing.build();
+
+  java.lang.Object execute(Vector self, long index) {
```

Comment:
What is it with these unnecessarily qualified names? Oo
```suggestion
 Object execute(Vector self, long index) {
```